buildSrc/src/main/kotlin/io/spine/internal/dependency/CheckStyle.kt

```diff
@@ -30,5 +30,6 @@ package io.spine.internal.dependency
 // See `io.spine.internal.gradle.checkstyle.CheckStyleConfig`.
 @Suppress("unused")
 object CheckStyle {
+    // We can't migrate just a version to a TOML file.
```

Comment:
This comment needs some context and the link to a broader explanation somewhere. It's hard to understand when you open this code.

We have the same comment in the `Pmd` object. It suggests having one place which tells a little story about what we do in these version-only objects.

Reply:
I mixed things up here. We are allowed to store bare versions in TOML.